### DISCUSSION TITLE
Questions for our AMA on Dec 10

### DISCUSSION THREAD

**Discussion by actionmancan** [containers/podman#24766] [General]
Hi,

We will be having an AMA on Dec 10. If you have any questions you would like answered leave them here.

Blog post with meeting link: https://blog.podman.io/2024/12/podman-bootc-podman-desktop-ama/

**Comment by TomSweeneyRedHat** [comment]
1. How does the move affect Red Hat's commitment to the tools?
2. Is there a transition period, and when is that anticipated?
3. How will problems be reported to the project? Will they still be in GitHub, via Red Hat, or the CNCF? Once moved to CNCF, how will problem reporting be coordinated? Will there be a new central location?
4. Are the GitHub project locations changing?

**Comment by actionmancan** [comment]
How can companies other than Red Hat help maintain the project and contribute to the roadmap?

**Comment by omenos** [comment]
The `bootc` documentation mentions the plans of lifting the `ostree` commit requirement in the future.

1. What kind of changes should be expected?
2. How will they impact currently available workflows, from image generation to deployment to day2 ops?
3. Will it be possible to transition existing systems to the newer images as a regular update or will they need to be re-provisioned?
4. Will the Red Hat family of base images continue to use `ostree` as their foundation?

---

1. In the current sysadmin landscape there is a lot of tooling available for managing systems. What kind of workflow and mindset changes are going to be required for those using conventional tooling today? For example:
  a. Satellite with content management
  b. subscription-manager
  c. Ansible (+AAP)
2. Does the project plan to integrate more natively with other tooling or platforms for faster initial deployments? The current workflow involves `Containerfile` > Bootc Image > Image Builder > Platform Outputs.

**Comment by Zeglius** [comment]
Greetings, might be out of scope (more leaning towards bootc) but is there any tldr regarding how would podman integrate with the former to distribute/build systemd extensions, and how these would be distributed (maybe as OCI artifact(?), as an fully fledged container image(?)).
